app/services/serializer_version_chooser.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class SerializerVersionChooser
+  DEFAULT_API_VERSION = 'V2'
+
+  def self.call(interpolatable)
+    serializer = "V2::#{interpolatable}Serializer"
```

Comment:
```suggestion
    serializer = "V2::#{interpolatable.to_s.singularize.camelize}Serializer"
```

Tested this in IRB and seems to work fine passing in either `:move` symbol or `Move` class.

Reply:
Also the constant doesn't seem to be used - could you add that rather than `V2` here and on line 12 pls?

Reply:
Done